### PR TITLE
Template editing

### PR DESCRIPTION
**Pull request by PatrikKopkan**

### CODE REVIEW

File: gen_spec.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import requests
 import sys
-from tempfile import TemporaryDirectory
 from subprocess import call, check_output
-
-from jinja2 import Template
+from tempfile import TemporaryDirectory
+import requests
```
Comment (frenzymadness) [inline gen_spec.py:5]: There should be one blank line between imports from stdlib and third-party modules.

https://www.python.org/dev/peps/pep-0008/#imports

File: template.spec [hunk 2]
```diff
@@ -1,32 +1,26 @@
 %global pkgname {{ pypi['info']['name'] }}
 
-Name:           python3-%{pkgname}
+# todo: This is not valid specfile
```
Comment (frenzymadness) [inline template.spec]: Why? And how does a comment like that is useful?

**Review by frenzymadness — CHANGES_REQUESTED** [review]
Thank you. Just a few nitpicks. And the last commit should be fixup commit so it can be automatically squashed during the merge. https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html

File: template.spec (hunk 2 above)
Comment (PatrikKopkan) [reply template.spec]: forget to delete note.

**Review by frenzymadness — APPROVED** [review]

**Pull request merged**